src/components/views/settings/tabs/room/RolesRoomSettingsTab.tsx

```diff
@@ -416,7 +431,7 @@ export default class RolesRoomSettingsTab extends React.Component<IProps> {
             .filter(Boolean);
 
         // hide the power level selector for enabling E2EE if it the room is already encrypted
-        if (client.isRoomEncrypted(this.props.room.roomId)) {
+        if (this.state.isRoomEncrypted) {
             delete eventsLevels[EventType.RoomEncryption];
         }
```

Comment:
This looks like it'll cause a UI flicker, where a row disappears after all rooms are checked, this is undesirable, we should not show UI until we are sure it is relevant.